src/decisionengine/framework/modules/de_logger.py

```diff
@@ -1,19 +1,29 @@
 """
 Logger to use in all modules
 """
+import copy
 import os
 import logging
 import logging.handlers
+import logging.config
+import structlog
+import decisionengine.framework.modules.logging_configDict as configDict
 
 MB = 1000000
 
-def set_logging(log_level,
-                file_rotate_by,
-                rotation_time_unit='D',
-                rotation_interval=1,
-                max_backup_count=6,
-                max_file_size=200 * MB,
-                log_file_name='/tmp/decision_engine_logs/decision_engine_log'):
+logger = structlog.getLogger("decision_engine")
+logger = logger.bind(module=__name__.split(".")[-1])
+
+
+def set_logging(
+    log_level,
+    file_rotate_by,
+    rotation_time_unit="D",
+    rotation_interval=1,
+    max_backup_count=6,
+    max_file_size=200 * MB,
+    log_file_name="/tmp/decision_engine_logs/decision_engine_log",
```

Comment:
Change the name to decision_engine.log or decisionengine.log, to have the same extension of the worker's logs?

Reply:
Sure. I can do that.

Reply:
Done.